airflow/triggers/temporal.py

```diff
@@ -84,5 +95,5 @@ class TimeDeltaTrigger(DateTimeTrigger):
     DateTimeTrigger class, since they're operationally the same.
     """
 
-    def __init__(self, delta: datetime.timedelta):
-        super().__init__(moment=timezone.utcnow() + delta)
+    def __init__(self, delta: datetime.timedelta, **kwargs):
+        super().__init__(moment=timezone.utcnow() + delta, **kwargs)
```

Comment:
Since the only valid kwarg is `exit_task` let’s spell it out directly; it’s friendlier for editors.

Reply:
sure